### ISSUE TITLE
LambdaCase

### ISSUE THREAD

**Issue by pharpend** [chrisdone/hindent#57]
First, here's the starting code:

```haskell
foo = \case
  Just bar -> baz
  Nothing -> quux
```

@gibiansky's style is converting this to

```haskell
foo = \case
  Just bar = baz
  Nothing = quux
```

Which is not valid code.

**Comment by pharpend** [comment]
@chrisdone's style does the same thing:

```haskell
foo =
  \case
    Just bar = baz
    Nothing = quux
```

**Comment by chrisdone** [comment]
It's due to the regrettable change in HSE to lump `->` and `=` in the same type.

**Comment by chrisdone** [comment]
Fixed in #61.
